pithos/StationsDialog.py

```diff
@@ -38,7 +38,9 @@ def __init__(self, pithos, *args, **kwargs):
         self.model = pithos.stations_model
         self.worker_run = pithos.worker_run
         self.quickmix_changed = False
-        self.searchDialog = None
+        self.searchDialog = SearchDialog.SearchDialog(self.worker_run)
+        self.searchDialog.set_transient_for(self)
```

Comment:
Once the `__init__()` func is fixed as mentioned above. `SearchDialog(self.worker_run, transient_for=self)`